### ISSUE TITLE
Completely broken on Firefox 108

### ISSUE THREAD

**Issue by TeryVeneno** [rafaelmardojai/firefox-gnome-theme#504]
FGT has completely stopped working on firefox 108. I use Firefox Developer Edition on my machine and found that around update 108.0b4 the theme made firefox unusable. I'm currently on 108.0b8 and the issues have persisted. I also tested on the nightly build of firefox and found the same issues.

![Screenshot from 2022-11-30 12-44-18](https://user-images.githubusercontent.com/104284259/204884324-45ad2241-d5a0-4224-b637-22a1ec9d9de5.png)

The toolbar merged with the url bar and navigation controls making them unclickable though I can still input a url. The right side hamburger menu has been merged with the close button and no longer works, but the window can still be closed. There are about 10 tabs open but none of them are visible except for the tab I'm on. And the extensions overflow menu and new tab buttons have become the second and third bars in place of the bookmarks and tabs that were there before. I wish I had more info, but I'm really lost as to what could have changed in one update that broke the theme so much.

**Comment by ferlanero** [comment]
I can confirm this problem in Firefox 108.0. I have had to completely remove firefox-gnome-theme in order to use Firefox again

**Comment by ruizlenato** [comment]
same here.

**Comment by emansom** [comment]
🖐🏻 also here.

Firefox 108 just got released and is being pushed to distros, soon this will affect everyone.

**Comment by emansom** [comment]
@rafaelmardojai @lunakurame @CleoMenezesJr

**Comment by rafaelmardojai** [comment]
I'm aware of this, but sadly I'm not sure if I will be able to work on the fix this week, possibly I do it on the weekend.

So for now I might disable my Flatpak's auto updates to avoid the issue. 

Pd: @emansom be aware than Luna is not longer involved on the project, so she may not be interested in receiving notifications about this.

**Comment by TimestaticYT** [comment]
Just noticed this issue too. I this installed on asahi and hopes it gets fixed soon. Luckily I don't use Linux as my main OS but when this comes into the flatpak it will probably cause a lot of trouble

**Comment by somepaulo** [comment]
> soon this will affect everyone

Yup. Just updated FF on Arch and it became borked. Had to rename the `~/.mozilla/firefox/xxxxxxxx.default/chrome/` folder until there's a fix. Didn't realize I'd forgotten how FF actually looks )

**Comment by Hannor-Smith** [comment]
I have this problem too. Have to reinstall my snap firefox and use the default theme for now. Hope it get fixed soon.

**Comment by gtsiam** [comment]
Also run into this after the 108 update etc. etc.

Just wanted to link to #467, since this and that issue are the same.

**Comment by witalihirsch** [comment]
I found the problem, you can solve it manually. see my last [commit](https://github.com/witalihirsch/Mono-firefox-theme)

**Comment by mkhl** [comment]
> I found the problem, you can solve it manually. see my last [commit](https://github.com/witalihirsch/Mono-firefox-theme)

thanks, it does! made a pr out of your fix: #513

**Comment by Ashdemai** [comment]
Thanks for the update!

**Comment by TheKrafter** [comment]
Was having the same issue, but reinstalled (with `scripts/install-by-curl.sh` from when i originally cloned the repo) and it is back to normal.

**Comment by Movisivi** [comment]
Broken on Nightly again, come on dude

**Comment by Ashdemai** [comment]
> Broken on Nightly again, come on dude

Don't "come on dude" him when he does it for free and you are using a build that should be expected to have breakages. You can let him know, but your tone of writing should be reevaluated.

**Comment by gtsiam** [comment]
@Movisivi I just checked using the latest nightly build - 110.0a1 (2023-01-06, from heftig's nighty repo for arch)

And uh... it's not broken.

At least not that I could find. If you find breakage, please respectfully open a new issue explaining what you did, what happened and what you expected. I will assume you've pulled the latest version and went over the [requirements](https://github.com/rafaelmardojai/firefox-gnome-theme#required-firefox-preferences) already.

**Comment by CleoMenezesJr** [comment]
> Broken on Nightly again, come on dude

Thank you for reporting this issuw in such a friendly way. If you feel that you are a dissatisfied customer, please contact Customer Service and ask for a full refund of the product's value.

**Comment by emansom** [comment]
> > Broken on Nightly again, come on dude
> 
> Thank you for reporting this issue in such a friendly way. If you feel that you are a dissatisfied customer, please contact Customer Service and ask for a full refund of the product's value.

Quite the refund there. 🤗
